samples/sampler/stories/default/MediaOverlay.js

```diff
@@ -10,7 +10,7 @@ const prop = {
 	textAlign: ['start', 'center', 'end'],
 	videos: {
 		Sintel: 'http://media.w3.org/2010/05/sintel/trailer.mp4',
-		'Big Buck Bunny': 'http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4',
+		'Cosmos_Laundromat': 'http://media.xiph.org/cosmoslaundromat/Pilot_Trailer_Cosmos_Laundromat.mp4',
```

Comment:
@dongsuwon Do you have any reason to use underscore? 'Cosmos_Laundromat' 
All others use 'Cosmos Laundromat'

Reply:
There is no reason. It's my mistake. I removed underscore.